.github/workflows/ci-scripts.yml

```diff
@@ -49,7 +49,7 @@ jobs:
             base: "7.0"
             deps: adsupport
 
-          - os: ubuntu-20.04
+          - os: ubuntu-latest
```

Comment:
As we discussed in person, maybe we should simply bump to 22.04, so it is still tested with older-ish software.

Moving to a container-based deploy would be the best option in the long run, but probably also harder to implement now.